app/validation/ValidatedActions.scala

```diff
@@ -5,22 +5,19 @@ import java.util.UUID
 import auth.TokenSecurity
 import configuration.GraphQLConfiguration
 import controllers.routes
-import graphql.codegen.IsTransferAgreementComplete.{isTransferAgreementComplete => itac}
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, Request, Result}
-import services.GetConsignmentService
+import services.{ConsignmentService, TransferAgreementService}
 
 import scala.concurrent.ExecutionContext
 
 abstract class ValidatedActions() extends TokenSecurity with I18nSupport {
   implicit val ec: ExecutionContext
   val graphqlConfiguration: GraphQLConfiguration
 
-  private val isTransferAgreementCompleteClient = graphqlConfiguration.getClient[itac.Data, itac.Variables]()
-
   def consignmentExists(consignmentId: UUID)(f: Request[AnyContent] => Result): Action[AnyContent] = secureAction.async {
     implicit request: Request[AnyContent] =>
-    val getConsignmentService = new GetConsignmentService(graphqlConfiguration)
+    val getConsignmentService = new ConsignmentService(graphqlConfiguration)
```

Comment:
Maybe rename to val consignmentService to match the renamed class

Reply:
Done, thanks.